### ISSUE TITLE
TensorFlow版本

### ISSUE THREAD

**Issue by byq-luo** [BatchClayderman/ForestAdvisor#1]
TensorFlow==2.*？

**Comment by yueryang** [comment]
Hi, thank you for your focus and your question. 

Here is the result of running ``pip show tensorflow``. 

```
C:\WINDOWS\system32>pip show tensorflow
Name: tensorflow
Version: 2.10.0
Summary: TensorFlow is an open source machine learning framework for everyone.
Home-page: https://www.tensorflow.org/
Author: Google Inc.
Author-email: packages@tensorflow.org
License: Apache 2.0
Location: d:\program files\python\lib\site-packages
Requires: absl-py, astunparse, flatbuffers, gast, google-pasta, grpcio, h5py, keras, keras-preprocessing, libclang, numpy, opt-einsum, packaging, protobuf, setuptools, six, tensorboard, tensorflow-estimator, tensorflow-io-gcs-filesystem, termcolor, typing-extensions, wrapt
Required-by:
```

Actually, I am currently finding out adapted tensorflow versions to make this repo adaptable to as many tensorflow versions as possible. If it is possible, I want to know your tensorflow version and whether the repo can be run. 

Since this repo is migrated and updated, there may be some bugs. We are still testing the ``plot_model`` function and checking whether the training and testing processes are correct.

**Comment by byq-luo** [comment]
> Hi, thank you for your focus and your question.
> 
> Here is the result of running `pip show tensorflow`.
> 
> ```
> C:\WINDOWS\system32>pip show tensorflow
> Name: tensorflow
> Version: 2.10.0
> Summary: TensorFlow is an open source machine learning framework for everyone.
> Home-page: https://www.tensorflow.org/
> Author: Google Inc.
> Author-email: packages@tensorflow.org
> License: Apache 2.0
> Location: d:\program files\python\lib\site-packages
> Requires: absl-py, astunparse, flatbuffers, gast, google-pasta, grpcio, h5py, keras, keras-preprocessing, libclang, numpy, opt-einsum, packaging, protobuf, setuptools, six, tensorboard, tensorflow-estimator, tensorflow-io-gcs-filesystem, termcolor, typing-extensions, wrapt
> Required-by:
> ```
> 
> Actually, I am currently finding out adapted tensorflow versions to make this repo adaptable to as many tensorflow versions as possible. If it is possible, I want to know your tensorflow version and whether the repo can be run.
> 
> Since this repo is migrated and updated, there may be some bugs. We are still testing the `plot_model` function and checking whether the training and testing processes are correct.

The model is successfully saved.
Failed saving the model image. Details are as follows.
module 'pydot' has no attribute 'InvocationException'
Please press enter key to retry (Remain: 4).
Failed saving the model image. Details are as follows.
module 'pydot' has no attribute 'InvocationException'
Please press enter key to retry (Remain: 3).
Failed saving the model image. Details are as follows.
module 'pydot' has no attribute 'InvocationException'
Please press enter key to retry (Remain: 2).
Failed saving the model image. Details are as follows.
module 'pydot' has no attribute 'InvocationException'
Please press enter key to retry (Remain: 1).
Failed saving the model image. Details are as follows.
module 'pydot' has no attribute 'InvocationException'
Please press enter key to continue.


My environment is Windows11.There is a problem with the training of the preservation of the training, and the PIP Install Pydot Graphviz has not solved my problem.Thanks.

**Comment by yueryang** [comment]
Thank you for your log. This problem is actually an issue caused by the function ``plot_model``, which is just for showing a flow chart of the model. You can still save your model and the loss history. 

This issue is not caused by my repo. You may need to refer to [https://github.com/keras-team/keras/issues/19040](https://github.com/keras-team/keras/issues/19040), which is from the keras official.
